Makefile.am

```diff
@@ -41,6 +41,8 @@ swupd_SOURCES = \
 	src/lib/strings.h \
 	src/lib/sys.c \
 	src/lib/sys.h \
+	src/lib/tp.c \
```

Comment:
I'm not a fan of such short file names since they are not descriptive, I'd prefer the file to be called _threadpool.c_, is really not that long